OpenRA.Game/Input/Hotkey.cs

```diff
@@ -28,7 +28,12 @@ public static bool TryParse(string s, out Hotkey result)
 
 			Keycode key;
 			if (!Enum<Keycode>.TryParse(parts[0], true, out key))
-				return false;
+			{
+				int c = 0;
```

Comment:
It would be nice if you could remove that `= 0`, too. It's redundant. Sorry for the hassle.